src/components/sections/About.section.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import AboutAirBalloon from '../../assets/about/AboutAirBalloon.svg';
+import AboutSection from '../../assets/about/AboutMeSection.svg';
+
+const About: React.FC = () => {
+    const [offsetY, setOffsetY] = useState(0);
+
+    const handleScroll = () => {
+        setOffsetY(window.scrollY * 0.45); // Handles the speed at which the balloon moves
+    };
+
+    useEffect(() => {
+        window.addEventListener('scroll', handleScroll);
+
+        return () => window.removeEventListener('scroll', handleScroll);
+    }, []);
+    return (
+        <div className="relative bg-gradient-to-b from-peachPuff to-brightUbe pb-10 pt-48">
+            {/* MAIN SVG (Background) */}
+            <div className="relative left-[-4%] w-[115%] overflow-hidden">
+                <img src={AboutSection} alt="Billboard" className="block" />
+            </div>
+            {/* About HawkHacks Content */}
+            <div className="absolute left-1/2 top-[70%] w-2/3 -translate-x-1/2 -translate-y-1/2 transform text-white ">
+                <h1 className="mb-5 text-center text-4.5xl uppercase drop-shadow-md lg:text-6.5xl xl:text-7.5xl">
+                    About Hawkhacks
+                </h1>
+                <div className="space-y-10 text-center">
+                    <p>
+                        HawkHacks came out of a desire to give everyone an equal
+                        opportunity to get into tech, whether that be
+                        programming, networking, researching, learning, or
+                        teaching.
+                    </p>
+                    <p>
+                        Join hundreds of students across Canada (and across the
+                        world) in a 36 hour period of exploration, creativity,
+                        and learning!
+                    </p>
+                    <p>
+                        Remember, you don’t have to be a pro to participate -
+                        show up with ten years or ten minutes of experience (oh
+                        yeah, and a great attitute too!)
+                    </p>
+                </div>
+            </div>
+            {/* Hawk Air Balloon */}
+            <img
+                src={AboutAirBalloon}
+                alt="Billboard"
+                className="absolute left-[70%] w-[20%]"
+                style={{ top: `calc(30% + ${offsetY}px)` }} // Move balloon with scroll
```

Comment:
I would try to use translate property instead of offset property in this case, if that's doable (not confident hhh)

Why is CSS translate often recommended for animations over position?

CSS translate is often recommended for animations because it uses the GPU, which is more efficient at rendering graphics and animations. This results in smoother animations and less CPU (Central Processing Unit) usage, which can improve the overall performance of a webpage.

Reply:
Didn't know that either! That's pretty sweet. Fixed it! :)